content/scalability/docs/control-plane.md

```diff
@@ -240,4 +244,50 @@ spec:
 
 - This FlowSchema captures all list event calls made by service accounts in the default namespace.
 - The matching precedence 8000 is lower than the value of 9000 used by the existing service-accounts FlowSchema so these list event calls will match list-events-default-service-accounts rather than service-accounts.
-- We're using the catch-all PriorityLevelConfiguration to isolate these requests. This bucket only allows 13 inflight requests to be used by these long-running list event calls. Pods will start to receive 429s as soon they try to issue more than 13 of these requests concurrently. 
+- We're using the catch-all PriorityLevelConfiguration to isolate these requests. This bucket only allows 13 inflight requests to be used by these long-running list event calls. Pods will start to receive 429s as soon they try to issue more than 13 of these requests concurrently.
+
+## Use Shared Informers 
+
+When building controllers and automation that integrate with the Kubernetes API you will often need to get information from Kubernetes resources. If you poll for these resources regularly it can cause a significant load on the API server.
```

Comment:
@rothgar We know this (list vs watch) is a major cause of API server latency/unresponsiveness from the customer tickets we get. Can we change the phrasing a bit to make the reader treat this recommendation with more urgency/seriousness.  Maybe a line or two on the ramifications of "significant load on the API server", and how it can lead to API server latency/slowness/unresponsiveness will help.  Also, perhaps we can callout that this is a common cause of various API server issues/latencies that we see .

Reply:
I added a top level section with details. LMK what you think

Reply:
LGTM. Thank you for adding this section!